### DISCUSSION TITLE
Image Stretch Issue still exist. Not resolved as per #702

### DISCUSSION THREAD

**Discussion by mjiskinng** [exceljs/exceljs#1798] [Q&A]
Excel Image stretches. Please provide permanent fix or guide me what steps to follow. I am using it inside SPFx (SharePoint Framework) to generate the Excel from a template excel file and store generated Excel in a document library on SharePoint 

![ImageStrech](https://user-images.githubusercontent.com/3298209/126703654-04b12bc7-7449-475e-977b-af49b94dc09d.JPG)

**Comment by skypesky** [comment]
怎么说呢,你的问题是啥啊,兄弟.

**Reply by mjiskinng** [reply]
Hi @skypesky  i am facing image stretch issue as per screenshot. i want to know how to solve it.

**Reply by Siemienik** [reply]
@mjiskinng please provide the basic information, which ExcelJS version are you using ;). 
Additionally, the information on how to reproduces that bug will be enormously helpful. I mean a code/repo/sandbox/jsfiddle/gist/whatever else.
Finally, it is much easiest to investigate the bug if you will upload the XLSX files that were read and/or saved.

Kindly regards

**Comment by mjiskinng** [comment]
Hi @sebikeller i will check the version but i installed the latest one user npm install
Excel file attached
[F-HOM-DEC-GAR-36 - 2 Wall Hanging Plant Terrarium Glass.xlsx](https://github.com/exceljs/exceljs/files/6991993/F-HOM-DEC-GAR-36.-.2.Wall.Hanging.Plant.Terrarium.Glass.xlsx)

**Comment by meilechwieder** [comment]
@mjiskinng how was your issue fixed in the end, I’m facing similar issue

**Reply by mjiskinng** [reply]
I use ExcelScripts and MS Flow to  solve the issue
